components/TestSubmissionsResults/TestSubmissionsResults.module.scss

```diff
@@ -0,0 +1,27 @@
+﻿@import '../../pageStyles/constants';
+
+.listItem{
+  font-family: $softwireFont;
+  font-size: $contentFontSize;
+  color: $softwireWhite;
```

Comment:
and this is really styling the text not the list item right?
So I'd have a class on that paragraph instead, and add these styles to that.

Reply:
changed p tag to h2 and added class